fcceeConfig/fccReconstruction.xml

```diff
@@ -181,28 +187,28 @@
       <parameter name="NBunchtrain" type="int" value="3"/>
       <!--Integration time per subdetector: at the moment 10us (from ALICE?) - 3369ns (Physics is at 1st BX) -->
       <parameter name="Collection_IntegrationTimes" type="StringVec" >
-	VertexBarrelCollection        6631
-	VertexEndcapCollection        6631
+	VertexBarrelCollection        380
```

Comment:
The parameter Collection_IntegrationTimes can go with the global parameters of the group Overlay, now that it is common to the 91 and 365 GeV

Reply:
fixed, thanks